### PR TITLE
Added HT with binary tree(queue)

### PR DESCRIPTION
**Pull request by selestiel**

### CODE REVIEW

File: HT_BinaryTree/functions.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include "functions.h"
+
+
+void show_tree(node* head)
+{
+    std::queue<node*> list;
+    list.push(head);
```
Comment (smay1613) [inline HT_BinaryTree/functions.cpp]: what if head is nullptr?

File: HT_BinaryTree/functions.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    while(true){
+    int vect = list.size();
```
Comment (smay1613) [inline HT_BinaryTree/functions.cpp]: please use the meaningful names

File: HT_BinaryTree/main.cpp [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+int main()
+{
+    node *head;
+    head = new node(1,nullptr,nullptr);
+    head->left = new node(2,nullptr,nullptr);
+    head->right = new node(3,nullptr,nullptr);
+    head->left->left = new node(4,nullptr,nullptr);
+    head->left->right = new node(5,nullptr,nullptr);
+    head->right->left = new node(6,nullptr,nullptr);
+    head->right->right = new node(7,nullptr,nullptr);
```
Comment (smay1613) [inline HT_BinaryTree/main.cpp]: you'll have the memory leak here

File: HT_Bracket_Stack/functions.cpp [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                else break ;
+            }
+            if(text[i] == ')')
+            {
+                if(brackets.top() == '(')
+                {
+                    brackets.pop();
+                    continue ;
+                }
+                else break ;
```
Comment (smay1613) [inline HT_Bracket_Stack/functions.cpp]: else statement is useless here

Comment (selestiel) [reply HT_Bracket_Stack/functions.cpp]: fixed

File: HT_BinaryTree/main.cpp (hunk 3 above)
Comment (selestiel) [reply HT_BinaryTree/main.cpp]: as i get it leak due to nullptr? changed to zero

Comment (smay1613) [reply HT_BinaryTree/main.cpp]: where is the delete?

**Pull request closed without merging**